aws-android-sdk-iot/src/main/java/com/amazonaws/mobileconnectors/iot/AWSIotMqttManager.java

```diff
@@ -210,6 +215,24 @@ public boolean isMetricsEnabled() {
     /** Override value for System.currentTimeInMillis.  Used for unit testing reconnect logic. */
     private Long unitTestMillisOverride;
 
+    /**
+     * Sets the usernameFields map.
+     *
+     * @param usernameFields usernameFields map
```

Comment:
naming - consider changing the name to 'setUserMetaData'

Reply:
Done